src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -518,7 +655,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
 
     entries.find(_.isFailure) match {
       case Some(failure) => failure
-      case _ => Success(entries.flatMap(_.get))
+      case _ =>
+        // Coalesce all the CallStartResults into one all-encompassing ExecutionStartResult.
+        val zero = ExecutionStartResult(startEntries = Set.empty)
```

Comment:
The Scala collections API uses `z` for this zero value, Scalaz actually spells it out as `zero` in `Monoid`.

http://eed3si9n.com/learning-scalaz/Foldable.html

Happy to name it whatever you like.